packages/backend/src/helpers/flow-templates.ts

```diff
@@ -86,13 +86,25 @@ export async function createFlowFromTemplate(
     for (let i = 1; i < steps.length; i++) {
       const step: TemplateStep = steps[i]
       validateAppAndEventKey(step, flowName)
+      // replace all parameters with {{user_email}} to the current user email
+      const updatedParameters = structuredClone(step?.parameters ?? {})
+      for (const [key, value] of Object.entries(updatedParameters)) {
+        // ignore objects e.g. conditions because nothing to replace inside for now
+        if (typeof value === 'string') {
+          const substitutedValue = String(value).replaceAll(
```

Comment:
nit: but if it's already of type string, u dont need to cast it to a string again

Reply:
thanks for the catch